src/doc/trpl/installing-rust.md

```diff
@@ -1,24 +1,24 @@
 % Installing Rust
 
 The first step to using Rust is to install it! There are a number of ways to
-install Rust, but the easiest is to use the `rustup` script. If you're on Linux
-or a Mac, all you need to do is this (note that you don't need to type in the
-`$`s, they just indicate the start of each command):
+install Rust, but first we ought to go over the officially supported architecture's.
 
-```bash
-$ curl -sf -L https://static.rust-lang.org/rustup.sh | sudo sh
-```
+* Windows (7, 8, Server 2008 R2)***
+* Linux (2.6.18 or later, various distributions), x86 and x86-64
+* OSX 10.7 (Lion) or greater, x86 and x86-64
 
-If you're concerned about the [potential insecurity][insecurity] of using `curl
-| sudo sh`, please keep reading and see our disclaimer below. And feel free to
-use a two-step version of the installation and examine our installation script:
+We extensively test Rust on these platforms, there are plans to support Android
+in the future, that documentation will be public as it becomes available.
```

Comment:
@steveklabnik perhaps just "there are plans to support *additional platforms* in the future", since that leaves the door open for whatever we decide to support, (plus is shorter as well)?